### ISSUE TITLE
Link to Discord channel doesn't get updated

### ISSUE THREAD

**Issue by brunogirin** [codebar/planner#1226]
## Description of the issue 📄
When setting up a virtual workshop and editing the workshop later, the Discord channel URL doesn't get updated even though the new version is visible in the edit screen.

## Screenshots 📷
The view screen:
![Annotation 2020-03-18 193555](https://user-images.githubusercontent.com/216028/77002277-260fe780-6953-11ea-82ab-a3b35520804c.png)

The edit screen (also note that on this screen it's called "Slack channel link" instead of "Discord channel link"):
![Annotation 2020-03-18 193644](https://user-images.githubusercontent.com/216028/77002278-26a87e00-6953-11ea-83ae-eccd741edf4f.png)

## Steps to fix 🛠
Steps to reproduce:
- Create a virtual workshop with a Discord link
- Realise that you made a mistake and update the Discord link in the workshop details
- Click save: the old link is visible in the view but not the new one
- Click edit: the new link is visible in the form

## To do 📋

* [ ] Claim this issue (comment below, or assign yourself if you are part of the codebar org)
* [ ] Fork and clone the repository
* [ ] Update the relevant files. Follow the steps to fix section in this issue.
* [ ] Commit your changes as one commit. Use the title of this issue as your commit message
* [ ] Submit a pull request
* [ ] Mention this issue in the PR description by including it's number
* [ ] Have your pull request reviewed & merged by a codebar team member

**Comment by despo** [comment]
@brunogirin the discord link is static and not meant to be updated. The slack channel link maps to the channel name (so if you press #channel_name it takes you to it in Slack).

https://github.com/codebar/planner/blob/master/config/locales/en.yml#L629

Is there any need to update that?

**Comment by brunogirin** [comment]
@despo the main need is for people like me who make mistakes when setting it up and need to update it later. I discovered when setting it up today that Discord invitations have a 1 day expiry unless you specify otherwise hence why I wanted to update the link with a non-expirable invitation.

**Comment by despo** [comment]
@brunogirin the specific link does not expire so it shouldnt be a problem. This just avoid the need for anyone to have to figure out how to create an invitation as it's always there.

<img width="719" alt="Screen_Shot_2020-03-18_at_21_30_44" src="https://user-images.githubusercontent.com/159200/77009448-da176f80-695f-11ea-83c8-41b461c08313.png">

However I just noticed that I did use the wrong link in the How to run a virtual workshop document  which I guess may be what caused the confusion?
